etc/p3-config/p3-config.yml

```diff
@@ -649,20 +650,51 @@ p3_image_fedora_coreos:
     os_distro: "fedora-coreos"
     os_version: "30"
 
-# Latest CentOS image
+# Latest CentOS 8
 p3_image_centos:
+  name: "CentOS8.2"
+  elements:
+    - "centos"
+    - "epel"
+    - "selinux-permissive"
+    - "alaska-extras"
```

Comment:
I had issues building the image using ``alaska-extras`` DIB element in its current form. Did you have success there?